store/ivy.xml

```diff
@@ -104,5 +104,7 @@
   <dependency org="zimbra" name="zm-native" rev="latest.integration"/>
   <dependency org="org.xmlunit" name="xmlunit-core" rev="2.3.0"/>
   <dependency org="ant-contrib" name="ant-contrib" rev="1.0b3" />
+  <dependency org="com.google.javascript" name="closure-compiler" rev="v20180204"/>
```

Comment:
Create tickets to include them as part of zm-build., I assume they go into /opt/zimbra/lib/jars

Reply:
Created ZCS-4808